src/components/transactions/AssetInput.tsx

```diff
@@ -148,28 +148,28 @@ export const AssetInput = <T extends Asset = Asset>({
             // eslint-disable-next-line
             inputComponent={NumberFormatCustom as any}
           />
-
+          {value !== '' && !disabled && (
```

Comment:
This should `!disableInput` instead of `!disable`

`disableInput` is passed in as true in the collateral repay and swap modals.

Reply:
Fixed 🙏